cfht/config/processCcd.py

```diff
@@ -0,0 +1,1780 @@
+import lsst.pipe.tasks.processCcd
```

Comment:
I thought a bit about whether including products irrelevant to jointcal (like the config and metadata) is helpful and eventually concluded that it is, because it serves to document the state of the pipeline when it was created.  But then where is the `packages.pickle` file?  Is there anything else missing?